README.md

```diff
@@ -204,11 +204,14 @@ is active and install the package, which builds the native library. There are
 two ways to install this:
 
 ```bash
+# Install dependencies:
+devenv sync
+
 # Install the release build, recommended:
-pip install --editable ./py
+uv pip install -v -e py
```

Comment:
Not too familiar with direnv but this command fails for me on a fresh relay clone. `.devenv/bin/ux` exists but `.devenv/bin` is not in my `$PATH`. I've tried calling `direnv allow` and `direnv reload`.

Reply:
Tried it again today with a fresh clone and it worked as expected 👍. Maybe I messed up the order of commands.